src/routes/chat/ChatHeader/ChatHeader.jsx

```diff
@@ -87,19 +93,50 @@ const ChatCreateOffer = ({ setOffer }) => {
 
 const ChatHeader = () => {
   const [offer, setOffer] = useState(true);
```

Comment:
Didn't catch this the last time, but for booleans please add a `is` prefix; i.e `isOffer, setIsOffer`